src/test/java/io/jenkins/plugins/analysis/core/model/TabLabelProviderTest.java

```diff
@@ -0,0 +1,83 @@
+package io.jenkins.plugins.analysis.core.model;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.Report;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests the class {@link TabLabelProvider}.
+ *
+ * @author Tobias Redl
+ */
+class TabLabelProviderTest {
+
+    private TabLabelProvider createTabLabelProvider(final String fileName) {
+        Issue issue = mock(Issue.class);
+        when(issue.getFileName()).thenReturn(fileName);
+
+        Report report = new Report();
+        report.add(issue);
+
+        return new TabLabelProvider(report);
+    }
+
+    @Test
+    void shouldReturnPackageNameJava() {
+        TabLabelProvider tabLabelProvider = createTabLabelProvider("Testfile.java");
+
+        assertEquals(tabLabelProvider.getPackageName(), Messages.Tab_Package());
```

Comment:
`assertThat(tabLabelProvider).hasPackageName(...)`

Reply:
I implemented Assertions, SoftAssertions and TabLabelProviderAssert with assertThat and hasPackageName + hasPackages